firedrake/cofunction.py

```diff
@@ -150,6 +150,14 @@ def equals(self, other):
             return True
         return self._count == other._count and self._function_space == other._function_space
 
+    def zero(self, subset=None):
+        """Set all values to zero.
```

Comment:
```suggestion
        """Set values to zero.
```
Since we can set a subset.